eslint.config.js

```diff
@@ -186,32 +170,43 @@ export default [
 				moduleTeardown: false,
 				url: false,
 				q: false,
-				jQuery: true,
-				sinon: true,
-				amdDefined: true,
-				fireNative: true,
-				Globals: true,
-				hasPHP: true,
-				isLocal: true,
-				supportjQuery: true,
-				originaljQuery: true,
-				$: true,
-				original$: true,
-				baseURL: true,
-				externalHost: true
+				jQuery: false,
+				$: false,
+				sinon: false,
+				amdDefined: false,
+				fireNative: false,
+				Globals: false,
+				hasPHP: false,
+				isLocal: false,
+				supportjQuery: false,
+				originaljQuery: false,
+				original$: false,
+				baseURL: false,
+				externalHost: false
 			}
 		},
 		rules: {
 			...jqueryConfig.rules,
-			strict: [ "error", "function" ],
```

Comment:
How did it work before?

Reply:
It was ignored because we hadn't set the source type. But, most of these files do not meet this requirement, so I left it out.